Source/engine/render/text_render.cpp

```diff
@@ -194,11 +194,10 @@ const OwnedClxSpriteList *LoadFont(GameFontTables size, text_color color, uint16
 	}
 
 	char path[32];
-	GetFontPath(size, row, "pcx", &path[0]);
+	GetFontPath(size, row, "clx", &path[0]);

```

Comment:
We need to some how inform users with older version of our data archives that they need to make sure to update fonts.mpq